native/src/seal/util/rlwe.cpp

```diff
@@ -364,14 +366,14 @@ namespace seal
             // then this code can be uncommented. Otherwise we do not actually
             // use the seed if it is provided to ensure the library is not
             // accidentally insecure.
-            // auto bootstrap_prng = seed.has_value() 
-            //      ? parms.random_generator()->create(seed.value())
-            //      : parms.random_generator()->create();
+            auto bootstrap_prng = seed.has_value() 
+                 ? parms.random_generator()->create(seed.value())
+                 : parms.random_generator()->create();
```

Comment:
Why is this exposed now? If you really wanted to do this, then the comment above should be updated to note that the user should be careful of using a seed.

Reply:
I mimicked what we had for asymmetric encryptions. I'll update both comments.

Reply:
Originally I added this code and I was hesitant to do so because I didn't want to make it easy to accidentally use an insecure setup. That was the only reason why I asked, but it does make sense to expose it since we have the same in the asymmetric case.